glowing_waffles/photometry.py

```diff
@@ -1,16 +1,13 @@
 from __future__ import print_function, division, absolute_import
 import numpy as np
-from astropy import units as u
-from astropy.stats import mad_std
-from astropy.io import fits
 from astropy.table import Table, join
 from photutils import aperture_photometry, CircularAperture, CircularAnnulus
 from .coordinates import convert_pixel_wcs
 
 __all__ = ['photutils_stellar_photometry']
 
 def photutils_stellar_photometry(ccd_image, sources, aperture_radius, inner_annulus
-                                 outer_annulus):
+                                 outer_annulus, gain, N_R, N_dark_pp):
```

Comment:
Could we switch these to keyword arguments, with defaults of `1.0` for `gain`, and zero for `N_R` and `N_dark_pp`?